runtime/compiler/runtime/J9CodeCache.cpp

```diff
@@ -822,31 +854,26 @@ J9::CodeCache::disclaim(TR::CodeCacheManager *manager, bool canDisclaimOnSwap)
    int32_t disclaimDone = 0;
 
 #ifdef LINUX
-   J9JavaVM * javaVM = jitConfig->javaVM;
-   PORT_ACCESS_FROM_JAVAVM(javaVM); // for j9vmem_supported_page_sizes
-
+   if ((uintptr_t)_smallPageAreaStart >= (uintptr_t)_smallPageAreaEnd)
+      return disclaimDone;
+   // Some of the warm code could have been written into the small page area.
+   // We don't want to disclaim warm code.
+   uint8_t *disclaimStart = (uintptr_t)_warmCodeAlloc > (uintptr_t)_smallPageAreaStart ? _warmCodeAlloc : _smallPageAreaStart;
```

Comment:
Don't we still need to ensure that `disclaimStart` is page aligned? How is `_warmCodeAlloc` guaranteed to be page aligned?

Reply:
You are correct. I'll fix it.

Reply:
Fixed.